src/protocol/http/org/apache/jmeter/protocol/http/curl/BasicCurlParser.java

```diff
@@ -166,7 +128,7 @@
         private String cookieInHeaders = "";
         private String filepathCookie="";
         private Authorization authorization = new Authorization();
-        private String cacert = "";
+        private String CACert = "";
```

Comment:
This is a normal variable, so we should start with a lower case char. So it would be `cACert`, which looks really strange. Therefore I would opt for `caCert`.